### PR TITLE
feat(ec2): capture ec2 metadata

### PR DESCRIPTION
**Pull request by idonava**

### CODE REVIEW

File: src/containers/ec2.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+module.exports.loadEC2Metadata = function loadEC2Metadata() {
+    if (currentEC2Labels) return Promise.resolve(currentEC2Labels);
+    const promises = [];
+    utils.debugLog('Loading EC2 metadata');
+    attributeToGet.forEach((attribute) => {
+        promises.push(axios.get(URL + attribute).then((response) => {
+            utils.debugLog(`Received response for ${attribute}: ${response.data}`);
+            if (response.status === 200) {
+                if (!currentEC2Labels) currentEC2Labels = {};
+                currentEC2Labels[`ec2.${attribute}`] = response.data;
```
Comment (ranrib) [inline src/containers/ec2.js]: I think we should do `aws.ec2.attribute`

File: src/containers/ec2.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const axios = require('axios');
+const utils = require('../utils');
+const eventIterface = require('../event');
+
+let currentEC2Labels = null;
+
+const URL = 'http://169.254.169.254/latest/meta-data/';
```
Comment (haimrait) [inline src/containers/ec2.js:7]: why is it a static ip?

File: src/containers/ec2.js (hunk 1 above)
Comment (idonava) [reply src/containers/ec2.js]: You're right.

File: src/containers/ec2.js (hunk 2 above)
Comment (idonava) [reply src/containers/ec2.js:7]: it's always the same IP to get the metadata in the running instance

File: src/containers/ec2.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+/**
+ * Load EC2 metadata and store it
+ * @returns {Promise} when resolved will contain the EC2 metadata
+ */
+module.exports.loadEC2Metadata = function loadEC2Metadata() {
+    if (currentEC2Labels) return Promise.resolve(currentEC2Labels);
+    const promises = [];
+    utils.debugLog('Loading EC2 metadata');
+    attributeToGet.forEach((attribute) => {
+        promises.push(axios.get(URL + attribute).then((response) => {
```
Comment (haimrait) [inline src/containers/ec2.js]: add a timeout to axios request, otherwise, it can take a lot of time until rejection

**Review by haimrait — APPROVED** [review]

**Pull request merged**